<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mahmoud" failures="0" tests="6" name="tests.EmailToFriendTest" time="5.627" errors="1" timestamp="2020-08-17T16:50:16 EET" skipped="4">
  <testcase classname="tests.EmailToFriendTest" name="screenshotOnFailure" time="0.001">
    <error message="Unresolved compilation problem: 
	FileUtils cannot be resolved
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
FileUtils cannot be resolved

at tests.TestBase.screenshotOnFailure(TestBase.java:131)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- screenshotOnFailure -->
  <system-out/>
  <testcase classname="tests.EmailToFriendTest" name="UserCanRegisterSuccessfully" time="5.626"/>
  <system-out/>
  <testcase classname="tests.EmailToFriendTest" name="UserCanEmailAFriend">
    <skipped/>
  </testcase> <!-- UserCanEmailAFriend -->
  <system-out/>
  <testcase classname="tests.EmailToFriendTest" name="UserCanSearchUsingAutoSuggest">
    <skipped/>
  </testcase> <!-- UserCanSearchUsingAutoSuggest -->
  <system-out/>
  <testcase classname="tests.EmailToFriendTest" name="RegisteredUserCanLogout">
    <skipped/>
  </testcase> <!-- RegisteredUserCanLogout -->
  <system-out/>
  <testcase classname="tests.EmailToFriendTest" name="UserCanSendEmail">
    <skipped/>
  </testcase> <!-- UserCanSendEmail -->
  <system-out/>
</testsuite> <!-- tests.EmailToFriendTest -->
